### DISCUSSION TITLE
Error: Navigation setup does not allow linking of pages or setting icons

### DISCUSSION THREAD

**Discussion by fx14** [requarks/wiki#5282] [Error / Bug Report] [answered]
I am new to Wiki.js. When setting up the navigation bar, i saw that i cannot select pages as targets. Nor can i select icons. 
How can I fix this? See screenshot: https://i.imgur.com/hedXi4H.png

Thanks a lot for your help,
Klaus

**Comment by jarrodlovas** [comment]
What installation method did you use because your navigation isn't looking correct. Can you please send a full-page screenshot?

**Comment by fx14** [comment, marked as answer]
I set it up manually on a Linux VM.

I only installed it during the last two days and am using the newest Wikijs version (2.5.281). 

Here is a full-page screenshots:
- of the Admin navigation page: https://imgur.com/cKo5ycE.png
- of the Wikijs setup: https://i.imgur.com/bGIL5IU.png
- of a full page: https://i.imgur.com/i2JSdFF.png

The full page looks OK, the admin panel works fine. Just the navigation does not have target pages, that's the only weird thing.

**Reply by jarrodlovas** [reply]
So if you want to link to target pages when adding a new navigation component you need to specify **link**.

1. Click the Add button and select Link:
![image](https://user-images.githubusercontent.com/47882472/168065215-431c262e-bb6d-48de-8b42-1cc35b09b537.png)

2. Click on your new navigation component and the new menu should appear that allows you to configure what the link does.
![image](https://user-images.githubusercontent.com/47882472/168065454-9b787ba9-7511-4997-a71f-acdf29ad2dee.png)

3. Click on the Target Type drop down many and select Page
![image](https://user-images.githubusercontent.com/47882472/168065774-3d68fdee-68cb-4c82-aa06-44e73887232e.png)

**Reply by fx14** [reply]
OK that solves it! Thanks a lot. I am new here, so i did not understand the difference of 'header' and 'link'. thanks a lot, you made my day!
